libsolidity/Compiler.cpp

```diff
@@ -197,7 +199,9 @@ void Compiler::appendFunctionSelector(ContractDefinition const& _contract)
 		m_context << callDataUnpackerEntryPoints.at(it.first);
 		eth::AssemblyItem returnTag = m_context.pushNewTag();
 		appendCalldataUnpacker(functionType->getParameterTypes());
-		m_context.appendJumpTo(m_context.getFunctionEntryLabel(functionType->getDeclaration()));
+		auto assemblyItem = m_context.getFunctionEntryLabel(functionType->getDeclaration());
+		//assemblyItem.setJumpType(eth::AssemblyItem::JumpType::IntoFunction);
```

Comment:
This, on the other hand, should not be a jump into a function, I would say (it is also likely to be optimized away).